addon/components/x-tree-branch.js

```diff
@@ -0,0 +1,16 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+
+export default class XTreeBranchComponent extends Component {
+  get loadMoreText() {
+	if(this.args.model.length !=0){
+		return this.args.model[0].loadMoreText;
+	}
+  }
+   
+  @action
+   loadingNext(modell){
```

Comment:
we can avoid defining `loadingNext` action in this component js file. Reason is, we are are writting anything special in it.
Have same method name onLoad means always onLoad.